sdk/python/feast/repo_config.py

```diff
@@ -19,10 +19,10 @@ class OnlineStoreConfig(NamedTuple):
 
 
 class RepoConfig(NamedTuple):
-    metadata_store: str
-    project: str
-    provider: str
-    online_store: OnlineStoreConfig
+    metadata_store: str = "./metadata_store"
+    project: str = "default"
```

Comment:
@oavdeev is this meant to be the GCP project?

Reply:
No I am pretty sure it's the Feast project.